environment_variables.md

```diff
@@ -0,0 +1,47 @@
+# Environment Variables
+
+| Name | Default | Description |
+| ------------------------------------|:---:| --- |
+| NGRAPH_CODEGEN | |
+| NGRAPH_COMPILER_DEBUGINFO_ENABLE | |
+| NGRAPH_COMPILER_DIAG_ENABLE | |
+| NGRAPH_COMPILER_REPORT_ENABLE | |
+| NGRAPH_CPU_BIN_TRACER_LOG | |
+| NGRAPH_CPU_CHECK_PARMS_AND_CONSTS | |
+| NGRAPH_CPU_CONCURRENCY | |
+| NGRAPH_CPU_DEBUG_TRACER | |
+| NGRAPH_CPU_EIGEN_THREAD_COUNT | |
+| NGRAPH_CPU_INF_CHECK | |
+| NGRAPH_CPU_NAN_CHECK | |
+| NGRAPH_CPU_TRACER_LOG | |
+| NGRAPH_CPU_TRACING | |
+| NGRAPH_CPU_USE_REF_KERNELS | |
+| NGRAPH_CPU_USE_TBB | |
+| NGRAPH_DECONV_FUSE | |
+| NGRAPH_DEX_DEBUG | |
+| NGRAPH_DISABLE_LOGGING | |
+| NGRAPH_DISABLED_FUSIONS | |
+| NGRAPH_ENABLE_REPLACE_CHECK | |
+| NGRAPH_ENABLE_SERIALIZE_TRACING | |
+| NGRAPH_ENABLE_TRACING | |
+| NGRAPH_ENABLE_VISUALIZE_TRACING | |
+| NGRAPH_FAIL_MATCH_AT | |
+| NGRAPH_GRAPH_REWRITE_RERUN_DYNAMIC_CHECK | |
+| NGRAPH_GTEST_INFO | |
+| NGRAPH_INTER_OP_PARALLELISM | |
+| NGRAPH_INTRA_OP_PARALLELISM | |
+| NGRAPH_MLIR | |
+| NGRAPH_MLIR_MAX_CYCLE_DEPTH | |
```

Comment:
`NGRAPH_MLIR_MAX_CYCLE_DEPTH ` is not used anymore. I will clean up sub-graph extraction code. Also, you missed `NGRAPH_MLIR_OPTIONS`

Reply:
:( There are so many env vars